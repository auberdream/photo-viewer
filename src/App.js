import React, { Component } from 'react';
import ImageSlideshow from './ImageSlideshow';
import Styles from './resources/styles/app.css'

class App extends Component {
  render() {
    return (
      <div className="app">
        <ImageSlideshow />
      </div>
    )
  }
}

export default App;
