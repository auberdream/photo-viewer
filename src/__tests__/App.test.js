import React from 'react';
import { shallow } from 'enzyme';
import App from '../App';
import renderer from 'react-test-renderer';

it('renders without crashing', () => {
  shallow(<App />);
})

it('renders correctly', () => {
  const tree = renderer
    .create(<App />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
