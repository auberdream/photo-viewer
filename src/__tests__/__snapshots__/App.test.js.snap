// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="app"
>
  <div
    className="main-container"
  >
    <nav>
      <div
        className="searchbar-container"
      >
        <div
          className="fake-textbox"
        >
          <img
            className="magnifying-glass"
            src="./images/magnifying-glass.png"
          />
          <input
            id="textbox"
            onChange={[Function]}
            placeholder="What are you looking for?"
            type="textbox"
          />
        </div>
        <div
          className="searchresults-container"
        />
      </div>
      <div
        className="button-container"
      >
        <input
          name="-1"
          onClick={[Function]}
          type="button"
          value="<"
        />
        <input
          id="shuffle"
          onClick={[Function]}
          type="button"
          value="S"
        />
        <input
          name="1"
          onClick={[Function]}
          type="button"
          value=">"
        />
      </div>
    </nav>
    <div
      className="slide"
    >
      <img
        src="https://lh3.googleusercontent.com/y6JGH-kgKKTChqCNEvtGUbSw3x7S4R1Y5uvCoj6desKVqIvBPyI6uwJz8q2ACUzx5FcFH9tgH4iltGa2G5XR8t98HqgZBuZHsXF9D-YUaq2Bu3P3i5gXPZFXSyv3pfEJkx2UGWCXEH2fuqLduyXyaaIMjqhe1-0InYIjTymy3PjSz5PrL6gu8FzyKGyQdlsFJJPXiboVxosEfWeqP_BBErSYHuBAwyrJuQeGgD9fg7Hx9lA9QUqFddsIqmAO-7giw_Oi3Yk6QK7HUI52HeIyvctMhvTQMYE68RZyV-ggzs3X_Me0F6wUP6PvjCu9DOTn3Huec4vneeIZrcnwEuL6wDYvwjEiDr92-HdTmTAbyN3V2ymN5lyBr0gHihGEJt0liuxaTKZZp0BKX44F4PAyxlngHtDpshfYO1y4h0BxRM_sR0g1M8b2HTsNHWP-ye3r5gQQRw8POHoNLWJpur2UTpM29L-nlK_qdCjl0yyp8RCpGpR1fso9qtVw276jKF-YllcnofBkzYFdTasa28ywrGupr3CC6_fn2t6A_z-qfMaUNo-8leo7EC-6d7rHgSi2=w1280-h658"
      />
      <div
        className="slide-information"
      >
        <div
          className="information-container"
        >
          <p
            className="photo-date"
          >
            2018-04-17
          </p>
          <p
            className="photographer-name"
          >
            Alan Berryhill
          </p>
          <p
            className="photo-tags"
          >
            Tags: person, outdoors
          </p>
        </div>
      </div>
    </div>
  </div>
</div>
`;
